### PR TITLE
New remote_e1 quirks file

### PR DESCRIPTION
**Pull request by javicalle**
Similar to the existing `remote_h1.py`.

Fix: #1809 
Fix: #1794

### CODE REVIEW

File: xiaomi/aqara/remote_e1.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        (TRIPLE_PRESS, RIGHT): {COMMAND: COMMAND_2_TRIPLE},
+        (LONG_PRESS, RIGHT): {COMMAND: COMMAND_2_HOLD},
+        (SHORT_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_SINGLE},
+        (DOUBLE_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_DOUBLE},
+        (TRIPLE_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_TRIPLE},
+        (LONG_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_HOLD},
+        # triggers when operation_mode == command
+        # known issue: it seems impossible to know which button being pressed
+        # when operation_mode == command
+        (ALT_SHORT_PRESS, BUTTON): {COMMAND: COMMAND_TOGGLE, ENDPOINT_ID: 1, ARGS: []},
```
Comment (dmulcahey) [inline xiaomi/aqara/remote_e1.py:223]: Are endpoint Id / args needed here?

Comment (javicalle) [reply xiaomi/aqara/remote_e1.py:223]: To be honest I do not know 🤷🏻‍♂️. I have only replicated the implementation of `remote_h1.py`:
* https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/xiaomi/aqara/remote_h1.py

Comment (javicalle) [reply xiaomi/aqara/remote_e1.py]: I have moved the file that was in a wrong path. 🤦🏻‍♂️ 
Removed estra args.

**Comment by javicalle** [comment]
Thanks

**Pull request merged**